### PR TITLE
update man page, example config and help message

### PR DESCRIPTION
**Pull request by mrbaseman**
The example config file, the help message and the man page were missing some options which are actually supported. This pull request suggests a few changes to complete them.

### CODE REVIEW

**Comment by mrbaseman** [comment]
some checks have failed, I'll redo

**Comment by mrbaseman** [comment]
Finally managed to pass the tests. Sorry for the noise, but the results at travis give different results than when I run them locally. Indentation in enums seems to be different depending on the environment in which the tests are run.

File: doc/openfortivpn.1 [hunk 1]
```diff
@@ -115,14 +126,24 @@ username = foo
 .br
 password = bar
 .br
+user-cert = /etc/openfortivpn/user-cert.pem
+.br
+user-key =/etc/openfortivpn/user-key.pem
+.br
```
Comment (adrienverge) [inline doc/openfortivpn.1]: This section is just an example, I prefer to keep it clear by just having a few options (not all).

File: etc/openfortivpn/config [hunk 2]
```diff
@@ -3,3 +3,12 @@ host =
...
 password =
+# set-dns =
+# set-routes =
+# pppd-use-peerdns =
+# trusted-cert =
+# ca-file =
+# user-cert =
+# user-key =
+# insecure-ssl =
+# cipher-list =
```
Comment (adrienverge) [inline etc/openfortivpn/config]: Same here, this is unneeded and I prefer keep conf files simple...

File: src/main.c [hunk 3]
```diff
@@ -95,7 +93,15 @@ USAGE \
...
-"      trusted-cert = othercertificatedigest6631bf...\n"
+"      trusted-cert = othercertificatedigest6631bf...\n" \
+"      set-dns = 1\n" \
+"      set-routes = 1\n" \
+"      pppd-use-peerdns = 1\n" \
+"      ca-file = /etc/openfortivpn/ca-bundle.pem\n" \
+"      user-cert = /etc/openfortivpn/user-key.pem\n" \
+"      user-key = /etc/openfortivpn/user-cert.pem\n" \
+"      insecure-ssl = 0\n" \
+"      cipher-list = HIGH:!aNULL:!kRSA:!PSK:!SRP:!MD5:!RC4\n"
```
Comment (adrienverge) [inline src/main.c]: same as above

**Comment by mrbaseman** [comment]
actually the undocumented config features were the decisive factor for me to make this pull request. I found the options by reading the source code, but I would expect them to be documented somewhere. I'm fine with removing some options from the sample config and from the help message, but I would keep them in the man page and refer to that place in the other files. Would that be ok for you?

**Comment by adrienverge** [comment]
> I'm fine with removing some options from the sample config and from the help message, but I would keep them in the man page and refer to that place in the other files. Would that be ok for you?

That's great.

Thanks for your contribution!

**Pull request merged**